server/src/main/java/com/cloud/storage/ImageStoreUploadMonitorImpl.java

```diff
@@ -395,6 +404,20 @@ public void doInTransactionWithoutResult(TransactionStatus status) {
                             VMTemplateVO templateUpdate = _templateDao.createForUpdate();
                             templateUpdate.setSize(answer.getVirtualSize());
                             _templateDao.update(tmpTemplate.getId(), templateUpdate);
+                            // For multi-disk OVA, check and create data disk templates
```

Comment:
i see the same commen t in three places. Is this eligible for a refactor as overloaded method?